tests/unit/test_vrf.py

```diff
@@ -0,0 +1,46 @@
+from backend.consensus.vrf import get_validators_for_transaction
+
+
+def list_of_dicst_to_set(list_of_dicts: list[dict]) -> set:
```

Comment:
@AgustinRamiroDiaz I'm not sure about these randomized tests but I guess there is no other way for these files

Reply:
I've added another test without randomization
I think that both of them are valuable, these tests are "black box" and the new one is "white box"